package.json

```diff
@@ -32,7 +32,7 @@
     "typescript": "^5.3.3"
   },
   "engines": {
-    "node": ">=16.13.0",
+    "node": ">=18",
```

Comment:
Why is this necessary?

Reply:
`Blob` interface only exists on node >= 18

Reply:
That's not a problem for a typing, it'll still be available in any supported typescript version and they'll just be unable to create something that matches the type (but that's not a breaking change in this library)

Reply:
fair enough, will drop the major